.gitattributes

```diff
@@ -1 +1,4 @@
 * text=auto eol=lf
+
+# Windows specific test files where we need CRLF endings
+crates/air_r_formatter/tests/specs/r/crlf/*.R text eol=crlf
```

Comment:
This sets it so that all files are _checked out_ with LF endings except this one directory, which is forced to have CRLF endings (even on Mac, and particularly on CI)